appveyor.yml

```diff
@@ -10,16 +10,20 @@ platform:
 configuration:
   - Release
 
+# Enable this if the build cache needs fixing
+environment:
```

Comment:
"Note: By default, saving cache is disabled in Pull Request builds. Use Save build cache in Pull Requests checkbox on the General tab of project settings if you need to save cache during PR builds. This setting is not exposed in YAML to prevent unauthorized cache modifications."